README.md

```diff
@@ -28,61 +26,60 @@ Modelled as a state machine, with the help of AWS step functions, Trapheus resto
 
 ## Table of Contents
 
-- * [➤ Pre-Requisites](#pre-requisites )
-- * [➤ Parameters](#parameters)
-- * [➤ Instructions](#instructions)
-- * [➤ Execution](#execution)
-- * [➤ How it Works](#how-it-works)
-- * [➤ Contributing to Trapheus](#contributing-to-trapheus)
-- * [➤ Contributors](#contributors)
+- - [➤ Pre-Requisites](#pre-requisites)
```

Comment:
@Madhura-saw did you make these changes? they seem to be unrelated to your PR.

Ideally i was expecting a diff only from this line https://github.com/intuit/Trapheus/pull/147/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R243

Can we please remove the unrelated changes to make this easier for review?

Reply:
No, I did not make any changes over there